dispatcher/src/main/java/io/zeebe/dispatcher/DispatcherBuilder.java

```diff
@@ -60,8 +60,8 @@ public DispatcherBuilder actorScheduler(final ActorScheduler actorScheduler) {
   }
 
   /** The max length of the data section of a frame */
-  public DispatcherBuilder maxFragmentLength(final ByteValue maxFragmentLength) {
-    this.maxFragmentLength = (int) maxFragmentLength.toBytes();
+  public DispatcherBuilder maxFragmentLength(final long maxFragmentLength) {
```

Comment:
A little nit picky, but we could do the same we did above and have the caller perform the cast to int.

Reply:
Very attentive. Will do that